.travis.yml

```diff
@@ -0,0 +1,3 @@
+language: ruby
+before_install:
+  - gem install bundler
```

Comment:
The code is version-agnostic so far, it should work with 1.9-2.2. That said, since 1.9.3 has officially reached end of life I'm fine with explicitly testing on only 2.0+.